[ADD] pos: how to add/modify customer notes from POS UI

LoredanaLrpz opened the pull request:
Task ID: 2622090

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/3955).

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+==================
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
```
```suggestion
:guilabel:`Customer Note` on the pad. Doing so opens a pop-up window on which you can add the
```
It is spelled without an -S on my runbot

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+Modify a customer note
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```
```suggestion
Select the product from the cart to modify an existing note and click :guilabel:`Customer Note`.
```

toaa-odoo replied:
```suggestion
Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```
To modify an existing note, select the product and click :guilabel:`Customer Note`.

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+=====================
+
+The customer notes appear on customers' receipts and invoices similarly to how they appeared on the
+cart, meaning under the related product.
```
```suggestion
cart, under their related product.
```

LoredanaLrpz replied:
I dismissed that suggestion, I think it looks fine the way it is

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+The customer notes appear on customers' receipts and invoices similarly to how they appeared on the
+cart, meaning under the related product.
+
+.. image:: customer_notes/notes-receipt.png
```
Not sure what purposes the notes serve if all they display is "Customer Note from SO". Maybe you should change it with something like "(This is where the customer note appears)"

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+==============
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
```
```suggestion
For instance, to provide cleaning and maintenance tips; they can also be used to track a
```

LoredanaLrpz replied:
I've added the first part, but left it as 2 sentences (with ".")

toaa-odoo commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
```
```suggestion
customer’s particular request, such as not wanting the product to be assembled for them.
```

LoredanaLrpz replied:
Good catch, like the suggestion

toaa-odoo reviewed (changes requested):
A few points to consider

LoredanaLrpz commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
+Doing so reopens the previous pop-up window. Alter the text and click :guilabel:`Add`.
```
reopens and alter are correct

LoredanaLrpz commented on content/applications/sales/point_of_sale/overview/customer_notes.rst:
```diff
@@ -0,0 +1,38 @@
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
+desired content for the note.
```
I changed it a bit, but the suggestion would imply that there's content already and that you add more to it, which is not the case here

xpl-odoo reviewed (commented):
Is there a reason for mixing plural and singular when discussing customer notes? I would use one or the other for the most part :)

- customer_notes.rst
- H1: Customer note
- H2 Add customer notes
- H3 Modify a customer note

LoredanaLrpz commented:
> Is there a reason for mixing plural and singular when discussing customer notes? I would use one or the other for the most part :)
> 
> * customer_notes.rst
> * H1: Customer note
> * H2 Add customer notes
> * H3 Modify a customer note

Hey @xpl-odoo, "Customer Notes" is the name of the feature/setting; I use it in the plural form when I refer to it as the feature.
A customer note would be the concept of leaving a note; here, it's no longer a feature, so I left it as a noun.


However, this being said, I changed the name of the .rst file and added an "a" in the titles to have it all in singular (except for the one in the last § because it made more sense to leave it in the plural form)

xpl-odoo commented:
@robodoo r+
